agent/log_streamer.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/buildkite/agent/v3/status"
 )
 
+const jobLogMaxSize = 1 << 30 // 1 GiB; fits in int32
```

Comment:
Personal opinion: I find the `1024 * 1024 * 1024` form (pronounced in my head as “kilobyte → megabyte → gigabyte”) easier to read/understand than the 1<<30 (even after lots of bit-shift-heavy side projects 😁).

The comment kinda fixes that, but then sometimes comments lie so it's nice to easily verify them.

Reply:
Makes sense, added.